src/views/assets/Asset/AssetMoreInformationEdit.vue

```diff
@@ -22,7 +22,7 @@ export default {
           fields: [
             'vendor', 'model', 'sn', 'cpu_model', 'cpu_count',
             'cpu_cores', 'cpu_vcpus', 'memory', 'disk_total',
-            'distribution', 'distribution_version', 'arch'
+            'distribution', 'distribution_version', 'arch', 'gpu_model'
           ]
         }
       },
```

Comment:
It looks like there is no significant difference found between this code snippet and the latest version of Vue. All parts appear to be in line with the current practices.